modules/virt-hot-plugging-cpu.adoc

```diff
@@ -19,3 +19,8 @@ You can increase or decrease the number of CPU sockets allocated to a virtual ma
 +
 If the VM is migratable, a live migration is triggered. If not, or if the changes cannot be live-updated, a `RestartRequired` condition is added to the VM.
 
+[NOTE]
+====
+Memory hot-plug changes are applied immediately. If the VM is migratable, a live migration is triggered. Otherwise, a restart is required. By default, a VM can be hot-plugged with up to 4 times its initial CPU or memory configuration. For example, a VM with 1 vCPU and 2 GiB memory can scale up to 4 vCPUs and 8 GiB before a restart is needed.
+You can adjust this behavior cluster-wide using the `maxHotplugRatio` parameter or override it per VM.
```

Comment:
```suggestion
```
rendering fine, but not sure why this shows as being a hard return; can't hurt to remote it